src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -596,7 +596,9 @@ class MacroAssembler: public Assembler {
   void NAME(Register Rs1, Register Rs2, const address dest) {                                            \
     assert_cond(dest != nullptr);                                                                        \
     int64_t offset = dest - pc();                                                                        \
-    guarantee(is_simm13(offset) && ((offset % 2) == 0), "offset is invalid.");                           \
+    guarantee(is_simm13(offset) && ((offset % 2) == 0),                                                  \
```

Comment:
bool is_even(intx x) instead of mod ?

Reply:
I'll change it, but note that I didn't modify the condition, just the error message!